enterprise/backend/test/metabase_enterprise/advanced_permissions/api/setting_test.clj

```diff
@@ -1,6 +1,7 @@
 (ns metabase-enterprise.advanced-permissions.api.setting-test
```

Comment:
just update the name of a helper func in this file and remove some unnecessary tests